src/components/VrxIcon/VrxIcon.vue

```diff
@@ -0,0 +1,86 @@
+
+<template>
+  <svg :id="'vrx-icon-' + icon" class="dark:text-white" :class="size" fill="none" :stroke="iconColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
in una pagina potrebbero esserci due icone uguale, se non serve non utilizzare l'id. Se serve fare delle manipolazioni sull'svg utilizzare un ref per salvarsi l'oggetto html e utilizzare quello